Dockerfile

```diff
@@ -1,23 +1,11 @@
 # python:2.7-alpine with GEOS, GDAL, and Proj installed (built as a separate image
 # because it takes a long time to build)
 FROM rapidpro/rapidpro-base
-ARG RAPIDPRO_VERSION
-ARG VCS_REF
-ARG BUILD_DATE
 ENV PIP_RETRIES=120 \
     PIP_TIMEOUT=400 \
     PIP_DEFAULT_TIMEOUT=400 \
-    C_FORCE_ROOT=1
-
-LABEL org.label-schema.build-date=$BUILD_DATE \
-        org.label-schema.name="RapidPro" \
-        org.label-schema.description="RapidPro allows organizations to visually build scalable interactive messaging applications." \
-        org.label-schema.url="https://www.rapidpro.io/" \
-        org.label-schema.vcs-ref=$VCS_REF \
-        org.label-schema.vcs-url="https://github.com/rapidpro/rapidpro" \
-        org.label-schema.vendor="Nyaruka, UNICEF" \
-        org.label-schema.version=$RAPIDPRO_VERSION \
-        org.label-schema.schema-version="1.0"
+    C_FORCE_ROOT=1 \
+    PIP_EXTRA_INDEX_URL="https://alpine-3.wheelhouse.praekelt.org/simple"
```

Comment:
Wheels automatically built & kept up to date with pyup.io using this repo https://github.com/praekeltfoundation/debian-wheel-mirror/